install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -95,6 +95,9 @@ global:
   imagePullSecrets:
     # - private-registry-key
 
+  # If true, create a headless service and endpoint for istio-pilot with the remotePilotAddress and
+  # sets the MeshConfig configmap discoveryAddress to 'istio-pilot.<namespace>'
+  remotePilotCreateSvcEndpoint: false
```

Comment:
Do we need this ? Can it be the default ?

Reply:
This is for the case where remotePilotAddress is an IP and we need to create a selectorless service and endpoint.  This would be the case when you don't have a common DNS between the 2 clusters and you want to enable controle-plane security, which always enables the `verify_subject_alt_name` option in envoy.

Reply:
To add... I couldn't figure out a safe way in helm templates to validate if pilotRemoteAddress is an IP so I added this variable to the chart.  IMO, I don't think we want to make it default true since as @Rigdon points out, there's a good chance folks' production setups would create a common DNS env like he's doing.